qiime2/sdk/tests/data/mapping_only_config.toml

```
@@ -1,4 +1,8 @@
 [parsl]
 
-[parsl.executor_mapping]
+[parsl.executor_mapping.dummy_plugin]
 list_of_ints = "test"
+concatenate_ints = "test"
```

Just so we can confirm the right executors are being associated with the right actions, these tests should reference different executors. 

```suggestion
concatenate_ints = "test2"
```